scheduler/test/cook/test/simulator.clj

```diff
@@ -671,37 +704,142 @@
+                                                  (assoc :job/group group-uuid)
+                                                  (assoc :job/priority (rand-int 100))
+                                                  (assoc-in [:job/resource 0 :resource/amount] (-> (rand-int num-cpus-per-host)
+                                                                                                   (* 0.5)
+                                                                                                   inc
+                                                                                                   double))
+                                                  (assoc-in [:job/resource 1 :resource/amount] (-> (/ num-mem-per-host 15)
+                                                                                                   rand-int
+                                                                                                   inc
+                                                                                                   (* 15)
+                                                                                                   (min num-mem-per-host)
+                                                                                                   double))
+                                                  (assoc :job/user job-user)
+                                                  (assoc :job/uuid (str (UUID/randomUUID)))
+                                                  (assoc :run-time-ms run-time-ms)
+                                                  (assoc :submit-time-ms submit-time-ms))))))
+                       resources-consumed (reduce (fn [resources job]
+                                                    (let [cpus (get-in job [:job/resource 0 :resource/amount])
+                                                          mem (get-in job [:job/resource 1 :resource/amount])
+                                                          run-time-ms (get job :run-time-ms)]
+                                                      (-> (* cpus mem run-time-ms)
+                                                          (+ resources))))
+                                                  0
+                                                  group-jobs)]
+                   (println {:num-jobs (count group-jobs)
+                             :resources {:consumed resources-consumed
+                                         :left (- resources-left resources-consumed)}})
+                   (recur (reduce conj! jobs group-jobs)
+                          (- resources-left resources-consumed)))))]
+    (println "created" (count jobs) "jobs.")
+    (-> (json/write-str jobs :key-fn #(-> % str (subs 1)))
+        (str/replace "\\/" "/")
+        (->> (spit (str "resources/in-jobs-4K.json"))))))
+
+(comment create-jobs-and-hosts
+  (create-hosts)
+  (create-jobs))
```

Comment:
Can we delete the `comment`?

Reply:
I have the comment in there as I do not want the functions to be marked as unused by the IDE (Curisve).

Reply:
You should definitely add a meta-comment to explain this comment.